runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -204,8 +241,17 @@ public void onGameTick(GameTick tick)
 			return;
 		}
 
-		final int currentTickCount = client.getTickCount();
-		taggedFish.entrySet().removeIf(entry -> entry.getValue() + config.timeoutDelay() < currentTickCount);
+		// nets that are not accepting fish are those currently not accepting or those who were
+		// not accepting in the previous tick (When a fish shoal is 2 tiles adjacent
+		// to a drift net and is moving to a net that is just being setup it will be denied
+		// even though the net is currently in the CATCHING status)
+		List<DriftNet> notAcceptingNets = NETS.stream()
+			.filter(net -> net.isNotAcceptingFish() || net.wasNotAcceptingFish())
+			.collect(Collectors.toList());
+
+		taggedFish.entrySet().removeIf(entry -> checkTagExpired(entry.getValue()) || checkFishNextToNet(entry.getKey(), notAcceptingNets));
```

Comment:
This line is a little long, can we break it off somewhere? Also i think we should `s/check/is/` in for both of these methods.